models/Tool/UUID/Dao.php

```diff
@@ -39,6 +39,19 @@ public function save()
         $this->db->insertOrUpdate(self::TABLE_NAME, $data);
     }
 
+    public function create()
+    {
+        $data = $this->model->getObjectVars();
+
+        foreach ($data as $key => $value) {
+            if (!in_array($key, $this->getValidTableColumns(static::TABLE_NAME))) {
+                unset($data[$key]);
+            }
+        }
```

Comment:
Maybe we can put that into a `private` helper function to avoid duplicate code.

Reply:
Agree, added helper function.

Reply:
@hstjep so that's the only remaining thing 😊 Thanks in advance!